src/attrs_strict/_type_validation.py

```diff
@@ -113,14 +123,12 @@ def _validate_elements(attribute, value, expected_type):
         # These base_types happen when you have string annotations and cannot
         # be used in isinstance.
         raise _StringAnnotationError()
-
-    if base_type != typing.Union and not isinstance(  # type: ignore
```

Comment:
I don't think this can be removed.

Reply:
This is not removed, I have just consolidated this with other if elses.